includes/Main.php

```diff
@@ -1287,7 +1287,7 @@ public function log_document_creation_to_order_notes( object $document, string $
 			$user = wp_get_current_user();
 
 			if ( ! empty( $user->user_login ) ) {
-				$user_note = sprintf(
+				$user_note = ' ' . sprintf(
 					/* translators: user login name */
 					__( '(User: %s)', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
Perhaps you could also consider removing the brackets from the string?

Reply:
Would it be good like this:

![image](https://github.com/user-attachments/assets/a64445d1-f8ff-4047-aa96-9713b1f15b23)

Or maybe this is better?

![image](https://github.com/user-attachments/assets/75c8f046-7e5e-4178-987f-e2fbf261db0b)

Reply:
Either approach works for me; I just wanted to suggest that we might be able to skip passing them to the translation function.